src/dmd/hdrgen.d

```diff
@@ -3189,25 +3189,29 @@ public:
         }
     }
 
-    void parametersToBuffer(Parameters* parameters, int varargs)
+    void parametersToBuffer(ParameterList pl)
```

Comment:
Looks like this needs updating as well to a C++ friendly signature, or made extern(D).

Reply:
Does C++ code ever call it? And it's a POD struct.